url-detector/src/main/java/com/linkedin/urls/detection/UrlDetectorOptions.java

```diff
@@ -57,6 +57,11 @@ public enum UrlDetectorOptions {
    */
   HTML(27), //00011011
 
+  /**
+   * Checks for URLs against the full IANA scheme list for maximum possible detections
+   */
+  EXTENDED_IANA_DETECTION(31),
```

Comment:
Thanks I definitely think this is best defined as an option but again I'm wondering about the value selected here, it's not a single bit flag like 64 or 128 would be.  I don't think you're intending selection of this option to affect other options are you?

Reply:
I wasn't sure what you were doing with those and didn't see an example of setting multiple flags at once. I just picked random numbers for mine as placeholders. I'm totally cool with changing them so it doesn't affect other options. Would you like me to do that and resubmit?